pkg/plugins/common/kustomize/v1/scaffolds/internal/templates/config/manager/config.go

```diff
@@ -95,8 +95,8 @@ spec:
           periodSeconds: 10
         resources:
           limits:
-            cpu: 100m
-            memory: 30Mi
+            cpu: 200m
+            memory: 200Mi
```

Comment:
200Mi seems like a high limit. Thoughts on 100?
```suggestion
            memory: 100Mi
```

Reply:
Seems reasonable and I have forced push the changes.